DSharpPlus/Clients/DiscordClient.Dispatch.cs

```diff
@@ -908,7 +909,7 @@ internal async Task OnGuildSyncEventAsync(DiscordGuild guild, bool isLarge, JArr
 
         internal async Task OnGuildEmojisUpdateEventAsync(DiscordGuild guild, IEnumerable<DiscordEmoji> newEmojis)
         {
-            var oldEmojis = new ReadOnlyDictionary<ulong, DiscordEmoji>(guild._emojis);
+            var oldEmojis = guild._emojis.ToImmutableDictionary();
```

Comment:
I think this is my fault, rather than making this immutable it should just go back to `new ConcurrentDictionary<ulong, DiscordEmoji(guild._emojis);`